compiler/rustc_span/src/symbol.rs

```diff
@@ -113,8 +114,6 @@ symbols! {
         Gen:                "gen", // >= 2024 Edition only
         Try:                "try", // >= 2018 Edition only
 
-        // NOTE: When adding new keywords, consider adding them to the ui/parser/raw/raw-idents.rs test.
```

Comment:
I added this here below because someone appending new keywords is probably likely to notice it down here than up above. But I guess moving is fine too.